portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -48,7 +44,20 @@ private String convertToJsonUsingGson(List<String> messages) {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
+    // Get comments in datastore, by most recent order at the top.
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
```

Comment:
Hmm, I feel like "Comment" could be a class constant. That'd make it more clear that the string is intentionally identical between here and the doPost method (and helps avoid typos if you ever need to re-use it).

The same logic applies to the "timestamp" and "text" properties but it feels a bit cumbersome to have a class constant for each property that you want to store. So no need to create a constant for those now. But if you end up adding more properties (eg: author, tags, edits, etc) I'd be curious to discuss and hear your thoughts on how to keep all the property keys consistent and readable between the doPost and doGet methods.

Reply:
Actually, i just started looking at your next PR (https://github.com/zpChris/step/pull/15) and you did some restructuring there which is relevant to this discussion. So disregard this comment, I'll add my thoughts to that Cl